tendrl/commons/jobs/__init__.py

```diff
@@ -56,24 +59,37 @@ def _process_job(self, raw_job, job_key):
     def _acceptor(self):
         while not self.job_consumer_thread._complete.is_set():
             gevent.sleep(2)
-
             # TODO(team) replace below raw write with a "EtcdJobQueue" class
             try:
                 jobs = tendrl_ns.etcd_orm.client.read("/queue")
             except etcd.EtcdKeyNotFound:
                 continue
-                
             for job in jobs.children:
-                executed = False
-                if job.value is None:
+                try:
+                    raw_job = {"job_id": None,
+                               "integration_id": None,
+                               "run": None,
+                               "status": None,
+                               "parameters": None,
+                               "type": None,
+                               "node_ids": None,
+                               "request_id": None,
+                               "errors": None,
+                               "parent": None}
+                    job_id = job.key.split("/")[-1]
+                    result = self.read_job(job.key)
+                    result['job_id'] = job_id
+                    for item in result:
+                        raw_job[item] = result[item]
```

Comment:
Please validate if result actually contains only raw_job.keys() and nothing extra.

This is very important validation feedback for backend and API both